tools/governance/governance.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.IO;
+using System.Text;
+
+namespace governance {
+	struct ModuleInfo {
+		public string Name;
+		public string Hash;
+		public string Url;
+
+		public ModuleInfo (string name, string hash, string url)
+		{
+			Name = name;
+			Hash = hash;
+			Url = url;
+		}
+	}
+
+	public static class StringExtensions {
+		public static IEnumerable<string> SplitLines (this string item)
+		{
+			return item.Split (new char [] { '\n' }, StringSplitOptions.RemoveEmptyEntries);
+		}
+
+		public static IEnumerable<string> Split (this string item, char character)
+		{
+			return item.Split (new char [] { character }, 1, StringSplitOptions.None);
+		}
+	}
+
+	class EntryPoint {
+		static IEnumerable<ModuleInfo> GetModules ()
+		{
+			string currentDirectory = Environment.CurrentDirectory;
+
+			Xamarin.Bundler.Driver.RunCommand ("/usr/bin/xcrun", new string [] { "git", "submodule", "status" }, new string [] { }, out StringBuilder output, true, 0);

```

Comment:
Any particular reason to use `xcrun` to run `git`? That will always run the Apple-provided `git` (and not `git` installed by brew for instance).

Reply:
Because I knew where xcrun always lives. No good reason I guess.